frontend/src/views/DiagnosticTunnel/QualityMeasureSteps/QualityTotalStep.vue

```diff
@@ -89,22 +105,19 @@ export default {
         (x) => !!x
       )
     },
-    errorHelperFields() {
-      const fields = []
-      if (this.totalError)
-        fields.push(...["valueBioHt", "valueSustainableHt", "valueEgalimOthersHt", "valueExternalityPerformanceHt"])
-      if (this.totalMeatPoultryError) fields.push("valueMeatPoultryHt")
-      if (this.totalFishError) fields.push("valueFishHt")
-      if (this.totalFamiliesError) fields.push(...["valueMeatPoultryHt", "valueFishHt"])
-      return fields
-    },
   },
   methods: {
     updatePayload() {
       this.checkTotal()
       if (!this.hasError) this.$emit("update-payload", { payload: this.payload })
     },
+    updatePayloadFromComponent(componentPayload) {
```

Comment:
Je renommerais cette fonction en quelque chose qui rend évident que la source n'est pas n'importe quel composant externe, mais plus précisément le composant ErrorHelper. Par exemple `onErrorHelperChange` ou `onErrorHelperUpdate`.

Ceci à cause de la ligne `this.errorHelperUsed = true`